docs/samples/dashboards/METRICS-EXPORT-FLOW.md

```
@@ -0,0 +1,239 @@
+
+```bash
+# Ensure Grafana instance has the required label
+oc label grafana grafana -n llm-observability app=grafana
+
+# Deploy dashboards via Kustomize
+oc apply -k deployment/components/observability/dashboards
+```
+
+### Dashboard CRD Structure
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)